docs/src/intro.md

```diff
@@ -0,0 +1,57 @@
+# TiSpace Introduction
+
+## Summary
+
+In this project, we will build a system that supports developers to use the K8s Pod as a virtual machine.
+
+The system has the following features:
+
+- Flexible and user-friendly resource management platform
+- Highly available and schedulable disk services
+- Secure Container Runtime for Lightweight Virtual Machines
+- Delivers consistent performance as standard Linux containers
+
+
+## Motivation
+
+At PingCAP, our developers use a lot of virtual machines for our development and testing work. However, these virtual machines have poor performance and they are heavily overdraft. In addition, at PingCAP we do not have a good management platform to distribute and manage these VMs for developers. Developers manage and request these machine resources by asking each other. 
+
+To optimize these inefficient processes, we decided to develop a management platform to efficiently manage and allocate machine resources.
+
+## Detailed design
+
+The whole system uses [K8s] as the infrastructure, and deploys [ceph] clusters in [K8s] for disk high availability, and replaces the [K8s] container layer with [kata] to provide good virtualization capability.
```

Comment:
`replaces` may be `choose`. K8s supports a wide variety of container runtimes. cri-o with kata is one of them.